### PR TITLE
[ENG-25360] Create Edge Firewall, Edge Functions and Network List schemas

### PR DESCRIPTION
**Pull request by Vitor-Azion**

### CODE REVIEW

**Review by PatrickMenoti — APPROVED** [review]

File: networklist.yaml [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+        results:
+          $ref: '#/components/schemas/NetworkLists'
+        schema_version:
+          type: number
+      type: object
+    UpdateNetworkListsRequest:
+      properties:
+        name:
+          type: string
+          example: Network List criada pela API
```
Comment (vinigfer) [inline networklist.yaml:187]:
```suggestion
          example: Network List created using the API
```

File: networklist.yaml [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+          type: string
+      type: object
+    NetworkLists:
+      properties:
+        id:
+          type: integer
+          example: 1614
+        name:
+          type: string
+          example: Network List criada pela API
```
Comment (vinigfer) [inline networklist.yaml:117]:
```suggestion
          example: Network List created using the API
```

File: edgefirewall.yaml [hunk 3]
```diff
@@ -0,0 +1,333 @@
+openapi: 3.0.0
+info:
+  title: Azion API
+  version: "3.0"
+servers:
+- url: https://api.azionapi.net/
+paths:
+  /edge_firewall:
```
Comment (vinigfer) [inline edgefirewall.yaml:8]: Should this file be on this PR? It's not mentioned on the PR title, and it's already being added on another PR: https://github.com/aziontech/azionapi-openapi/pull/33

**Review by rogerclaus — COMMENTED** [review]
I have read the CLA Document and I hereby sign the CLA

**Review by rogerclaus — COMMENTED** [review]
recheck

**Comment by Vitor-Azion** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by trgalho** [comment]
recheck

**Review by rogerclaus — APPROVED** [review]

**Review by vinigfer — APPROVED** [review]

**Pull request merged**